packages/suite/src/actions/wallet/coinmarketP2pActions.ts

```diff
@@ -0,0 +1,66 @@
+import { COINMARKET_P2P } from '@wallet-actions/constants';
+import { Dispatch } from '@suite-types';
+import * as modalActions from '@suite-actions/modalActions';
+import invityAPI from '@suite-services/invityAPI';
+import { P2pProviderInfo, P2pQuote, P2pQuotesRequest } from 'invity-api';
+
+export interface P2pInfo {
+    country?: string;
+    suggestedFiatCurrency?: string; // optional field, fiat currency based on user's IP
```

Comment:
`state.wallet.settings.localCurrency` could be used
![image](https://user-images.githubusercontent.com/3729633/193290104-d614002d-d36e-4665-a98d-a7852beb241a.png)

Reply:
@IvaFingerova, could you please comment? Do we want to use the mentioned configuration property? In what way?

Reply:
Heya, I have no idea what that property does? @matejkriz 
In this integration, we are re-using the same logic as is in the buy feature

Reply:
@IvaFingerova That property shows the user's preference for the fiat currency (it is what user set in Suite Settings). 
I believe it makes sense to use it as suggested fiat currency instead guessing by IP address (which almost certainly does not give the correct value when you have Tor enabled).

Reply:
What is the inital value of this property? Does it always have a value?

Reply:
It always have a value, the initial value is simply `'usd'`

Reply:
So what do you think, @IvaFingerova? Should we at least use it as fallback value when `suggestedFiatCurrency` is not available for any reason? Currently `USD` fallback is hardcoded.

Reply:
Okie Dokie - fallback makes sense this way! Thanks guys 🙏

Reply:
Done, works like a charm!